src/js/views/patients/worklist/filters_views.js

```diff
@@ -52,13 +53,14 @@ const ClinicianDropList = Droplist.extend({
         attr: 'name',
       };
     });
-  },
-});
 
-const ClinicianClearButton = View.extend({
-  template: hbs`<button class="button-secondary worklist-list__clear-filter">{{ @intl.patients.worklist.filtersViews.clearClinicianFilter }}</button>`,
-  triggers: {
-    'click': 'click',
+    const currentUser = Radio.request('bootstrap', 'currentUser');
+
+    this.lists.unshift({
+      className: 'picklist__group',
+      collection: new Backbone.Collection([currentUser]),
+      attr: 'name',
```

Comment:
Just noticing.. I _think_ `attr` can move up into `picklistOptions` and removed here and in the `groups.map`